packages/paste-core/components/typography/src/Heading.tsx

```diff
@@ -0,0 +1,108 @@
+import * as React from 'react';
+import {Text} from '@twilio-paste/text';
+
+export type asTags = 'h1' | 'h2' | 'h3' | 'h4' | 'h5' | 'h6' | 'div' | 'label' | 'span';
+export type HeadingStyle =
+  | 'headingStyle10'
+  | 'headingStyle20'
+  | 'headingStyle30'
+  | 'headingStyle40'
+  | 'headingStyle50'
+  | 'headingStyle60';
+
+interface Heading {
```

Comment:
Let's make it a habit to export these interfaces as well so that they can be extended by consumers.

Reply:
No idea if it's possible, but I created a ticket to automate linting this

Reply:
Making the Heading props exportable means I have to change the name back to HeadingProps because the exportable name Heading is taken by the component export. Going that route for now.